tests/glvis_driver.py

```diff
@@ -72,31 +72,31 @@ def compare_images(baseline_file, output_file, expect_fail=False):
             print("[FAIL] Differences were not detected in the control case.")
         else:
             print("[PASS] Images match.")
-    print("       actual ssim = {}, cutoff = {}".format(ssim, cutoff_ssim))
+    print(f"       actual ssim = {ssim}, cutoff = {cutoff_ssim}")
     return ssim >= cutoff_ssim if not expect_fail else ssim < cutoff_ssim
 
 # Function to test a given glvis command with a variety of key-based commands.
 # Not currently in use.
 def test_case(exec_path, exec_args, baseline, t_group, t_name, cmd):
-    print("Testing {0}:{1}...".format(t_group, t_name))
+    print(f"Testing {t_group}:{t_name}..."))
```

Comment:
Extra `)` here ❗

Reply:
oops, I used a vim macro and it wasn't correct 😅. thanks